eve/main.yml

```diff
@@ -114,6 +115,22 @@ stages:
           usePTY: true
           haltOnFailure: false
 
+  unit_tests:
+    worker:
+      type: kube_pod
+      path: eve/workers/pod-unit-tests/pod.yaml
+      images:
+        docker-unit-tests: eve/workers/pod-unit-tests
+    steps:
+      - Git: *git_pull
+      - ShellCommand:
+          name: Run all UI unit tests
+          workdir: build/ui
+          command: >
+            npm install --production -no-cache --no-save -q &&
```

Comment:
`--no-cache` ?

Reply:
It's a "workaround" to avoid some "access denied" problems

Reply:
I think the comment was more about `-` vs `--`.
In the Unix world, long options tends to use `--` (like `--production` and `--no-save`) whereas short one use a single `-`.

Maybe `npm` is different, but this looks like a typo.